manifests/repo/yum.pp

```diff
@@ -33,15 +33,48 @@
     }
   }
 
+  case $facts['os']['name'] {
+    'RedHat': {
+      if $facts['os']['release']['major'] == '8' {
+        exec { 'Enabling Codeready-Builder repo':
+          command => 'dnf config-manager --set-enabled codeready-builder-for-rhel-8-x86_64-rpms',
+          path    => ['/usr/bin', '/usr/sbin',],
+        }
```

Comment:
This will trigger for each run. Maybe you can use some `unless` to make this idempotent?

Reply:
yes, will add that one.